File: FurnitureAPI/FurnitureAPI/Models/AdminPurchase.cs
using FurnitureAPI.Data.Models;
using System.Collections.Generic;
using System;

namespace FurnitureAPI.Models
{
    public class AdminPurchase
    {
        public string UserName { get; set; }
        public DateTime OrderDate { get; set; }
        public float Price { get; set; }
        public string Address { get; set; }
        public List<FurnitureWithTypeNameAndQuantity> Furnitures { get; set; }
    }
}

File: FurnitureAPP/app/src/main/java/com/example/furnitureapp/models/AdminPurchase.kt
package com.example.furnitureapp.models

import com.example.furnitureapp.datalayers.retrofit.models.FurnitureWithQuantityModel
import com.google.gson.annotations.SerializedName

data class AdminPurchase(
    val userName: String,
    val orderDate: String,
    val finalPrice: Float,
    val address: String,
    val furnitures: List<FurnitureWithQuantity>
)
